lib/roo_on_rails/routemaster/publisher.rb

```diff
@@ -0,0 +1,49 @@
+module RooOnRails
+  module Routemaster
+    class Publisher
+      attr_reader :model, :event
+
+      def initialize(model, event)
```

Comment:
I'd suggest dependency-injecting a `client: ::Routemaster::Client` here — otherwise people are going to stub like hell when testing publisherS?